requirements-test.txt

```diff
@@ -568,9 +622,9 @@ requests==2.26.0 \
     # via
     #   -r requirements.txt
     #   requests-kerberos
-requests-kerberos==0.12.0 \
-    --hash=sha256:5733abc0b6524815f6fc72d5c0ec9f3fb89137b852adea2a461c45931f5675e0 \
-    --hash=sha256:9d21f15241c53c2ad47e813138b9aee4b9acdd04b82048c4388ade15f40a52fd
+requests-kerberos==0.13.0 \
```

Comment:
@ejegrova 

Could you please separate those kerberos related changes and the amqp version bump into different commits?

The point is that the requests-kerberos update brings couple more dependencies. And it would be hard to recognize regressions.

Reply:
splitted the commits, check it please